src/pages/dashboard/sentry/SentryDetail.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import DashboardLayout from '../../../layouts/DashboardLayout';
+import { Collapse, Descriptions, Divider, List, PageHeader } from 'antd';
+import api from '../../../helpers/Api.js';
+import moment from 'moment';
+
+const { Panel } = Collapse;
+
+class SentryDetail extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+      id: this.props.location.pathname.split("/").pop(),
```

Comment:
Do not do this manually. Try to follow this > https://reacttraining.com/react-router/web/example/url-params